ext/sqlite3/sqlite3.c

```diff
@@ -1393,6 +1393,86 @@ PHP_METHOD(sqlite3stmt, readOnly)
 }
 /* }}} */
 
+/* bind parameters to a statement before execution */
+static int php_sqlite3_bind_params(php_sqlite3_stmt *stmt_obj) /* {{{ */
+{
+	struct php_sqlite3_bound_param *param;
+
+	if (stmt_obj->bound_params) {
+		ZEND_HASH_FOREACH_PTR(stmt_obj->bound_params, param) {
+			zval *parameter;
+			/* parameter must be a reference? */
+			if (Z_ISREF(param->parameter)) {
+				parameter = Z_REFVAL(param->parameter);
+			} else {
+				parameter = &param->parameter;
+			}
+
+			/* If the ZVAL is null then it should be bound as that */
+			if (Z_TYPE_P(parameter) == IS_NULL) {
+				sqlite3_bind_null(stmt_obj->stmt, param->param_number);
```

Comment:
@bohwaz Where are the `return_code` checks?

Reply:
Sorry, but I've already committed the checks (525d3ae858ec33a1e9fe72c339d053e65d252430), but this PR would remove them.

Reply:
You are right, I missed that, just copy/pasted my previous code sorry :)

Fixed now.